src/SerialLoops/Dialogs/BgmVolumePropertiesDialog.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Eto.Forms;
+using HaruhiChokuretsuLib.Util;
+using NAudio.Wave;
+using NAudio.Wave.SampleProviders;
+using SerialLoops.Controls;
+using SerialLoops.Lib.Items;
+using SerialLoops.Lib.Util.WaveformRenderer;
+using SerialLoops.Utility;
+using SkiaSharp;
+using System.IO;
+
+namespace SerialLoops.Dialogs
+{
+    public class BgmVolumePropertiesDialog : Dialog
+    {
+        private ILogger _log;
+        private WaveStream _wav;
+        private StackLayout _waveformLayout;
+        private long _waveLength;
+
+        public bool SaveChanges { get; set; } = false;
+        public BgmVolumePreviewItem VolumePreview { get; set; }
+        public SKGuiImage Waveform { get; set; }
+        public SoundPlayerPanel VolumePreviewPlayer { get; set; }
+
+        public BgmVolumePropertiesDialog(WaveStream wav, string title, ILogger log)
+        {
+            _log = log;
+            Title = title;
+            _wav = wav;
+            Waveform = new(WaveformRenderer.Render(wav, WaveFormRendererSettings.StandardSettings));
+            _waveformLayout = new()
+            {
+                Orientation = Orientation.Horizontal,
+                Items =
+                {
+                    Waveform,
+                }
+            };
+            _waveLength = wav.Length;
+            VolumePreview = new(wav);
+            VolumePreviewPlayer = new(VolumePreview, _log);
+
+            InitializeComponent();
+        }
+        
+        public void InitializeComponent()
+        {
+            Slider volumeSlider = new()
+            {
+                MinValue = 0,
+                MaxValue = 200,
+                Value = 100,
+                Orientation = Orientation.Vertical,
+            };
+            volumeSlider.ValueChanged += (obj, args) =>
+            {
+                _wav.Seek(0, SeekOrigin.Begin);
+                VolumePreview.SetVolume(volumeSlider.Value);
+                Waveform = new(WaveformRenderer.Render(VolumePreview.Provider, _waveLength, WaveFormRendererSettings.StandardSettings));
+                _wav.Seek(0, SeekOrigin.Begin);
+                _waveformLayout.Items.Clear();
+                _waveformLayout.Items.Add(Waveform);
+                VolumePreviewPlayer.Stop();
+            };
+
+            Button saveButton = new() { Text = "Save" };
+            Button cancelButton = new() { Text = "Cancel" };
+            saveButton.Click += (obj, args) =>
+            {
+                SaveChanges = true;
+                Close();
+            };
+            cancelButton.Click += (obj, args) =>
+            {
+                Close();
+            };
+
+            Content = new StackLayout
+            {
+                Orientation = Orientation.Vertical,
+                Spacing = 5,
```

Comment:
```suggestion
                Spacing = 5,
                Padding = 10,
```
This needs just a smidge of padding so the Pause/Stop buttons aren't giving the window controls a hug of death